src/Agent.Sdk/Util/PlatformUtil.cs

```diff
@@ -152,13 +152,33 @@ private static void DetectRHEL6()
             }
         }
 
+        private static string GetLinuxReleaseFilePath()
+        {
+            if (RunningOnLinux)
+            {
+                if (File.Exists("/etc/os-release"))
```

Comment:
Can these file paths be made as const?

Reply:
Replaced with a read-only array of strings, thanks!